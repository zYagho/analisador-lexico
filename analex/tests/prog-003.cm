int a;
float b;

int main(void){
  int x;
  float y;

  return(0);
}

