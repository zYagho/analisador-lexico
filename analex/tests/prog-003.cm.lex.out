INT
ID
SEMICOLON
FLOAT
ID
SEMICOLON
INT
ID
LPAREN
VOID
RPAREN
LBRACES
INT
ID
SEMICOLON
FLOAT
ID
SEMICOLON
RETURN
LPAREN
NUMBER
RPAREN
SEMICOLON
RBRACES
